src/Augmentation/ui/PlayerMultipliers.tsx

```diff
@@ -70,8 +70,14 @@ function MultiplierList(props: IMultiplierListProps): React.ReactElement {
               secondary={
                 <span style={{ display: "flex", alignItems: "center", flexWrap: "wrap" }}>
                   <BitNodeModifiedStats base={current} mult={bnMult} color={color} />
-                  <DoubleArrow fontSize="small" color="success" sx={{ mb: 0.5, mx: 1 }} />
-                  <BitNodeModifiedStats base={augmented} mult={bnMult} color={Settings.theme.success} />
+                  {current === augmented ? (
+                    <></>
+                  ) : (
+                    <>
+                      <DoubleArrow fontSize="small" color="success" sx={{ mb: 0.5, mx: 1 }} />
+                      <BitNodeModifiedStats base={augmented} mult={bnMult} color={Settings.theme.success} />
+                    </>
+                  )}
```

Comment:
Nice! One suggestion:

```suggestion
                  {current !== augmented &&
                    <>
                      <DoubleArrow fontSize="small" color="success" sx={{ mb: 0.5, mx: 1 }} />
                      <BitNodeModifiedStats base={augmented} mult={bnMult} color={Settings.theme.success} />
                    </>
                  }
```

(That might not lint right, might have to tweak parens.) The purpose is that false-y values get ommited from the React tree completely, as opposed to what you had where there'd be an empty node. Just a minor cleanliness thing.

Reply:
oh thats my bad i thought a random false would be worse than an empty fragment there :o